streaming/java/streaming-api/src/main/java/org/ray/streaming/jobgraph/LanguageType.java

```diff
@@ -0,0 +1,15 @@
+package org.ray.streaming.jobgraph;
+
+public enum LanguageType {
```

Comment:
```suggestion
public enum Language {
```

Reply:
I think 'LanguageType' is better.
![image](https://user-images.githubusercontent.com/23303886/72678967-992ce700-3ae5-11ea-9241-29b5a7fb54f5.png)

Reply:
Language sounds more nature.